rules/ask-for-help/rule.md

```diff
@@ -20,21 +20,16 @@ related:
 created: 2022-08-30T04:06:01.553Z
 guid: 113b1e11-6df4-4901-9e9f-bcc06b6fb352
 ---
-When you're working, you will encounter issues that block you from progressing, force you to do some googling, and investigation on how to move forward. These moments can be stressful, especially for junior developers and the question arises... **"When should I ask for help?"**
 
-<!--endintro-->
-
-As everyone's struggles are different and everyone's way of handling pressure differs, it's hard to provide a useful metric for when to ask for help. However, here's a general rule to follow... If you:
+Ideally, you should point out any problems you have with a work item during Sprint Planning. However, sometimes you think a PBI will be fine, but then you run into blocking issues during the sprint. Then, you are forced to do some googling, and investigation on how to move forward. These moments can be stressful, especially for junior developers and the question arises... **"When should I ask for help?"**
```

Comment:
This reads like you should wait for the sprint planning ceremony before you bring up that you're stuck. Try something like this:
"When you're working, you will encounter issues that block you from progressing, force you to do some googling, and investigation on how to move forward. These moments can be stressful, especially for junior developers and the question arises... **"When should I ask for help?"**

::: greybox
**Tip:** If you're taking on work which you know that you'll need help with ahead of time, make sure people know and and that you'll try your best but you'll be asking them questions. If people know ahead of time they're much more receptive to your questions.
:::
"

Reply:
It's crucial to mention Sprint Planning so people know to raise issues when they are assigned, not days later.

I've reworded it to make it clear that is specifically for when you are first assigned the issue though and not when you get blocked throughout.

Note that I didn't add the tip because raising issues in the sprint planning will be another rule that will be linked to this one in the first sentence, it just hasn't been created yet.

Please re-read to see if it is better for you.